protected/humhub/modules/space/modules/manage/views/default/index.php

```diff
@@ -26,7 +27,8 @@
         <?php $form = ActiveForm::begin(['options' => ['id' => 'spaceIndexForm'], 'enableClientValidation' => false]); ?>
 
         <?= SpaceNameColorInput::widget(['form' => $form, 'model' => $model]) ?>
-        <?= $form->field($model, 'description')->textarea(['rows' => 6]); ?>
+        <?= $form->field($model, 'description')->textInput(); ?>
+        <?= $form->field($model, 'about')->widget(RichTextField::class); ?>
```

Comment:
We have to make sure to post process the richtext input when saving the model, e.g. in order to attach files and do some other post processing. So when saving this form you need to call `RichText::postProcess($model->about, $model)`